typhos/positioner.py

```diff
@@ -269,7 +343,12 @@ def _define_setpoint_widget(self):
         if selection:
             self.ui.set_value = QtWidgets.QComboBox()
             self.ui.set_value.addItems(setpoint_signal.enum_strs)
-            self.ui.set_value.currentIndexChanged.connect(self.set)
+            self.ui.set_value.activated.connect(self.set)
```

Comment:
A small but very important fix, changing the `set` signal to `activated`.

Reply:
Should I point this out with a descriptive comment, or is that overboard?

Reply:
I'll go with the mantra of "If you think it might warrant a comment, and you can't decide, just make the comment"

Reply:
Sounds good to me 😁